server/controllers/finance/reports/cash/receipt.handlebars

```diff
@@ -1,5 +1,8 @@
 {{> head title="CASH.TITLE" }}
-
+<head>
+  <meta charset="utf-8" />
```

Comment:
This will result in two `<head>` tags.  It would be better to put this code in the `head.handlebars` partial that is added automatically.